mindsdb/integrations/handlers/bard_handler/README.md

```diff
@@ -0,0 +1,65 @@
+# Bard Handler
+The Bard ML handler integrates the Bard library from Google's Generative AI with MindsDB. You can use it to generate text completions with the Gemini Pro model for your existing text data.
+
+## Google Generative AI (Bard)
+Google Generative AI is a library that provides access to powerful language models for text generation. The Bard library, used in this handler, offers various models for generating text. More information about the Google Generative AI library can be found [here](https://github.com/GoogleCloudPlatform/generative-ai).
+
+*Note:* Ensure you have the necessary API key for accessing the Bard library.
```

Comment:
Please explain the difference between the API key required at ML engine creation time versus the one passed when creating the actual model. In the screenshot attached to the PR description you use two different API keys (which, by the way, you did not blur and so these are now compromised).

It is not clear what each one does. If possible, perhaps rename them to be more descriptive (e.g. `cloud_api_key` and `model_api_key`)

Reply:
yes will revoke that key

Reply:
btw both are the same keys don't have any difference